meeshkan/serve/mock/matcher.py

```diff
@@ -775,8 +777,8 @@ def truncate_path(path: str, o: OpenAPIObject, i: Request,) -> str:
 
 
 def match_request_to_openapi(
-    req: Request, r: Mapping[str, OpenAPIObject]
-) -> Mapping[str, OpenAPIObject]:
+    req: Request, specs: List[OpenAPISpecification]
+) -> List[OpenAPISpecification]:
```

Comment:
Maybe use `Sequence` instead of `List` if you're not modifying them?

Reply:
Done! I've changed it in the parameter, but the return type is still a `List` as it's needed (it's shuffled).